reservation_units/models.py

```diff
@@ -604,8 +604,13 @@ def check_reservation_overlap(self, start_time, end_time, reservation=None):
 
         return qs.exists()
 
-    def get_next_reservation(self, end_time: datetime.datetime, reservation=None):
-        from reservations.models import STATE_CHOICES, Reservation
+    def get_next_reservation(
+        self,
+        end_time: datetime.datetime,
+        reservation=None,
+        include_blocked: bool = True,
```

Comment:
So as below mentioned this could be `exclude_blocked`

Reply:
Fixed 👍